### DISCUSSION TITLE
QMenu does not show on v-touch-hold on QItem

### DISCUSSION THREAD

**Discussion by bigradish** [quasarframework/quasar#13414] [General - Components / Directives / etc] [answered]
### What happened?

When QMenu is contolled by a v-model, which is triggered on v-touch-hold on a QItem, the QMenu is just shown by a click on the QItem, not by a "touch hold".

### What did you expect to happen?

The QMenu is shown by a "touch hold" on the QItem.

### Reproduction URL

https://codepen.io/bigradish/pen/RwQGejM

### How to reproduce?

1. Go to the codepen.
2. Touch hold the "Single line item" QItem.

### Flavour

Vite Plugin (@quasar/vite-plugin)

### Areas

Components (quasar), Directives (quasar)

**Comment by pdanpdan** [comment]
QMenu shows by default on click. In your case you need to use `no-parent-event` to prevent this.
But you also need to show the menu on pointer event position - so you need to use `.show` method with the pointer event as param.

https://codepen.io/pdanpdan/pen/oNEzmMo?editors=1010

**Comment by bigradish** [comment]
Hi pdanpdan,

Thank you very much for your answer.
I have a question: how to let the popuped menu be closed automatically when the area outside the menu is clicked/tapped?

**Comment by pdanpdan** [comment, marked as answer]
You need to specify the `target` - I updated the codepen https://codepen.io/pdanpdan/pen/oNEzmMo?editors=1010
